pkg/runtime/deep_copy_generated.go

```diff
@@ -21,9 +21,39 @@ limitations under the License.
 package runtime
 
 import (
+	unversioned "k8s.io/kubernetes/pkg/api/unversioned"
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	reflect "reflect"
 )
 
+func DeepCopy_runtime_NoopDecoder(in NoopDecoder, out *NoopDecoder, c *conversion.Cloner) error {
```

Comment:
We don't want to have DeepCopy functions for those things. Can you please add a tag to those to avoid generating functions for those types (encoders, decoders, serializers, etc.)